content/guide/avalanche9000-motivation.mdx

```diff
@@ -0,0 +1,94 @@
+
+<YouTube id="eczkjrGG_As" />
+^^ placeholder
+
+## ACP-77
+
+### How Subnets Work Today
+
+![Subnets](/common-images/multi-chain-architecture/acp-77/subnets.png)
+
+The Avalanche network has a large set of validators who all participate in running and validating the Primary Network.
+
+The Primary Network consists of the C-Chain, P-Chain, and X-Chain, and is where most of the transactions on Avalanche are processed.
+
+In order to scale horizontally and offer more blockspace, a subset of validators can opt-in to validate an additional blockchain in parallel with the Primary Network, we call this a Subnet.
+
+### Subnets -> L1s
+With ACP-77 Subnets will become their own Layer 1 blockchain.
+
+![L1s](/common-images/multi-chain-architecture/acp-77/l1s.png)
+
+Instead of using subsets of validators from the Primary Network - we now use a new distinct set of validators.
+
+This change maintains the short time of finality and high throughput Avalanche Subnets are known for while also enabling a new set of features with options for completely customizable staking options, gas tokens, and governance.
+
+
+### Current Economic Barriers
+
+![Staking Requirements](/common-images/multi-chain-architecture/acp-77/subnets-staking-requirements.png)
+The Primary Network currently has a staking requirement of 2,000 AVAX.
+
+If we assume a price of $20 per AVAX, this would be a $40,000 staking requirement per validator.
+
+To launch a Subnet today, validators must meet the staking requirement for each subsequent Subnet they choose to validate. This is typically a native token to the Subnet.
+This model can quickly lead to a high economic barrier of entry in creating new Subnets as validators must meet the 2,000 AVAX requirement in addition to the staking requirements of any Subnets they want to validate.
+
+### Lowering Economic Barriers
+![L1 Staking Requirements](/common-images/multi-chain-architecture/acp-77/l1s-staking-requirement.png)
+
+This upgrade will remove the requirement to stake 2,000 AVAX on the Primary Network, and instead only require validators to stake on L1 with their native token.
+This will lower the economic barrier to entry, and enable more validators to participate in the network.
+
+
+### Current Running Costs
+![Subnet Running Cost](/common-images/multi-chain-architecture/acp-77/subnet-running-cost.png)
+
+Currently, validators must validate the Primary Network in addition to each Subnet they are validating. This can lead to higher running costs for validators as they must maintain machines that can handle the load of both networks.
+
+### Lower Running Costs
+![L1 Running Cost](/common-images/multi-chain-architecture/acp-77/l1-running-cost.png)
+With ACP-77, validators will only need to validate the L1. This will lead to lower running costs for validators as they will not need to maintain machines that can handle the load of both the Subnets and Primary Network.
```

Comment:
```suggestion

With ACP-77, validators will only need to validate the L1. This will lead to lower running costs for validators as they will not need to maintain machines that can handle the load of both the Subnets and the Primary Network.
```